### PR TITLE
Add changelog for v10.3.0-rc.1

### PR DESCRIPTION
**Pull request by pengdev**
### Summary of changes

This PR adds change log for v10.3.0-rc.1 release.

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.3.0-rc.1 January 26, 2022
+
+## Features ✨ and improvements 🏁
+* Refine viewport plugin's public APIs. ([#1069](https://github.com/mapbox/mapbox-maps-android/pull/1069))
+* Introduce metadata setter API for the legacy offline region. ([#1071](https://github.com/mapbox/mapbox-maps-android/pull/1071))
```
Comment (kiryldz) [inline CHANGELOG.md:9]: Can we add method name here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.3.0-rc.1 January 26, 2022
+
+## Features ✨ and improvements 🏁
+* Refine viewport plugin's public APIs. ([#1069](https://github.com/mapbox/mapbox-maps-android/pull/1069))
```
Comment (kiryldz) [inline CHANGELOG.md:8]: I guess actual changes list may be useful in release notes (but not here)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,24 @@
...
+* Introduce metadata setter API for the legacy offline region. ([#1071](https://github.com/mapbox/mapbox-maps-android/pull/1071))
+* Optimize zooming on terrain and globe. ([#1071](https://github.com/mapbox/mapbox-maps-android/pull/1071))
```
Comment (kiryldz) [inline CHANGELOG.md:10]: just curious - is it actually zoom optimized when indeed using 3D terrain + globe projection? For me it seems more to be optimize zooming when using 3D terrain _or_ globe projection but would be nice to confirm

Comment (pengdev) [reply CHANGELOG.md]: Updated to the change log entry to the original PR, I assume it's for both cases and combined use case as well.

**Review by yunikkk — APPROVED** [review]

**Review by kiryldz — APPROVED** [review]

**Review by ank27 — APPROVED** [review]

**Pull request merged**